planning/planning_validator/test/src/test_planning_validator_pubsub.cpp

```diff
@@ -55,7 +59,7 @@ class PubSubManager : public rclcpp::Node
 
 void spinSome(rclcpp::Node::SharedPtr node_ptr)
 {
-  for (int i = 0; i < 50; ++i) {
+  for (int i = 0; i < 10; ++i) {
```

Comment:
Need to make sure this spin time is enough for CICD. This works well in my environment.